content/modules/ROOT/pages/rhads-ai/rhads-ai.adoc

```diff
@@ -16,44 +15,44 @@ Learn more about link:https://www.redhat.com/en/products/ai[Red Hat AI,window='_
 == How RHADS enables AI tools for AI Engineers
 
 
-=== RHADS integrations with other tools
-Red Hat Advanced Developer Suite integrates with over 100 plugins, including many other integrations such as accessing Red Hat OpenShift Dev Spaces from the component's overview.
+=== {product_rhdh_name} integrations with other tools
+{product_rhdh_name} integrates with over 100 plugins, including many other integrations such as accessing Red Hat OpenShift Dev Spaces from the component's overview.
 Next, we'll explore key integrations for building AI applications and models including security best practices to build, deploy AI applications.
 
-=== RHDH integration with OpenShift AI
-Red Hat Developer Hub can integrate with OpenShift AI through Software Templates. These can be created to facilitate the model development lifecycle and ensure best practices are in place, such as security and scalability. It can provision Notebooks, create Serving Runtimes, create an inference server, Pipeline configurations, and more. Additionally, from the component's overview, AI Engineers can access many OpenShift AI features.
+=== {product_rhdh_name} integration with OpenShift AI
+Red Hat Developer Hub can integrate with OpenShift AI through Software Templates. These can be created to facilitate the model development lifecycle and ensure best practices are in place, such as security and scalability. It can be used by developers to provision Notebooks, create Serving Runtimes, create an inference server, Pipeline configurations, and more. Additionally, from the component's overview, AI Engineers can access many OpenShift AI features.
```

Comment:
they are not developers who will provision notebooks or serving runtimes. it'll be MLOPS/Data Scientists . I'd keep it as it is.